ivy_tests/test_ivy/test_functional/test_core/test_linalg.py

```diff
@@ -1017,12 +1025,13 @@ def test_svdvals(*, dtype_x, test_flags, backend_fw, fn_name, on_device):
 @handle_test(
     fn_tree="functional.ivy.tensordot",
     dtype_x1_x2_axis=_get_dtype_value1_value2_axis_for_tensordot(
-        available_dtypes=helpers.get_dtypes("numeric"),
+        available_dtypes=helpers.get_dtypes("valid"),
         min_num_dims=1,
         max_num_dims=5,
         min_dim_size=1,
         max_dim_size=10,
     ),
+    test_gradients=st.just(False),
 )
 def test_tensordot(*, dtype_x1_x2_axis, test_flags, backend_fw, fn_name, on_device):
```

Comment:
Given that we converted this function to a partial mixed function, we need to make some changes to the test. This is because the function will now use the primary implementation or the compositional one depending on the input, so we need to make sure that we generate a valid set of dtypes for each example. Here's the section in the [docs](https://unify.ai/docs/ivy/overview/deep_dive/ivy_tests.html#testing-partial-mixed-functions) explaining how to write the tests for partial mixed functions. Happy to help here if you find it a bit confusing :)